### ISSUE TITLE
Background Color Of Labels

### ISSUE THREAD

**Issue by GuittonHubert** [openglobus/openglobus#717]
Dear All,

Would it be possible to fill the  box of label with a background color, like in the example below? I suppose the answer is no.

![image](https://github.com/openglobus/openglobus/assets/61505725/cb849f5c-5c6e-48f9-9f8d-2258692d78f8)

The best location to handle this feature is certainly in LabelHandler.js but it is still a little bit cryptic to me. Is there a chance to get it? Thank you for considering this request.

Best regards,

Hubert

**Comment by Zemledelec** [comment]
No, it is not possible right now. It needs to be make some improvements with label handler.

**Comment by Zemledelec** [comment]
Now it is possible. I will think about it.
